packages/lib/src/components/OnlineBankingCZ/index.ts

```diff
@@ -7,11 +7,16 @@ const ICON = 'bankTransfer_IBAN';
 class OnlineBankingCZElement extends IssuerListContainer {
     public static type = 'onlineBanking_CZ';
 
+    private static termsAndConditions = {
```

Comment:
What is the benefit of using a static variable for this? 

We currently use static properties to define only the `type` and `defaultProps` , which kinda makes sense due to how we implement to get the payment method type and Component defaultProps in the library. But for this case, I don't understand the benefit

Reply:
I think it would make sense to use the static here because `termsAndConditions` doesn't change per `OnlineBankingCZElement` instance, it kinda belongs to the class itself.